tpm2tools/flush_active_handles.go

```diff
@@ -0,0 +1,40 @@
+package tpm2tools
+
+
+import (
+    "io"
+    "github.com/google/go-tpm/tpm2"
+    "github.com/google/go-tpm/tpmutil"
+)
+
+// FlushActiveHandles calls FlushContext() on all active handles within the 
+// TPM at io.ReadWriter rw. Returns nil if successful or TPM has no active
+// handles.
+func FlushActiveHandles(rw io.ReadWriter) error {
+    handles, err := activeHandles(rw)
+    if err != nil {
+        return err
+    }
+    for _, handle := range handles {
+        err = tpm2.FlushContext(rw, handle)
+        if err != nil {
+            return err
+        }
+    }
+    return nil
+}
+
+// activeHandles returns a slice of tpmutil.Handle objects of all handles
+// active within the TPM at ioReadWriter rw.
+func activeHandles(rw io.ReadWriter) ([]tpmutil.Handle, error) {
+    vals, _, err := tpm2.GetCapability(rw, tpm2.CapabilityHandles, ^uint32(0), 0x80000000)
```

Comment:
Here we discard the `moredata` return parameter. This is not correct if I'm reading the spec correctly. We should just check that `moredata` is false (as this should [_never_ return true](https://github.com/google/go-tpm-tools/blob/f14bb1799b8e2e113c8e15f1df230254d3249904/simulator/Object.c#L874) for tpm2.CapabilityHandles)

Reply:
Done.